app/logic/FlexibleApi.scala

```diff
@@ -56,11 +58,16 @@ class FlexibleApi(wsClient: WSClient) extends Loggable {
   }
 
   def restore(stack: FlexibleStack, user: User, contentId: String, snapshot: Snapshot): Attempt[String] = {
-    val userHeader = "X-GU-User" -> Json.stringify(Json.toJson(user))
+    val userString = Json.stringify(Json.toJson(user))
+
+    val userHeader = "X-GU-User" -> userString
+    val userHeaderBase64 = "X-GU-User-base64" -> Base64.getEncoder.encodeToString(userString.getBytes("UTF-8"))
+
+    val userHeaders = List(userHeader, userHeaderBase64)
```

Comment:
This won't be deployed until the API knows how to understand both and will use either is that correct? On that basis you should be safe to simply flip over from one to another. Although bear in mind that PROD restorer talks to both CODE and PROD APIs.

Reply:
Updated now; only sending the base64 encoded header for simplicity.

Reply:
@sihil are you happy for this to go?